okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -94,9 +94,9 @@
 
     static {
         dispatcher = new Dispatcher(executionExecutor);
-        dispatcher.setMaxRequests(256);
+        dispatcher.setMaxRequests(1024);
```

Comment:
I think it would be great to have a comment explaining why this is the _right_ number, i.e. what data do we need to look at to be confident this is somehow an optimal tradeoff and what are we trading off here.

(i.e. why shouldn't this be INT_MAX or 10)

Reply:
Talking to @j-baker it might actually make sense to rip out this entirely (and set it to INT_MAX), as we already have the concurrency limiter making sure that we don't overwhelm upstreams.